1. Made a responsive grid layout for last two movie poster sliders.

AlekseiKolesnikov opened the pull request:
I made a responsive grid layout for the last two movie/series sliders.  I also use an external library for reformating date.

AlekseiKolesnikov committed:
1. Made a responsive grid layout for last two movie poster sliders.

ShadowZilot commented on src/components/common/SliderGrid/SliderGrid.tsx:
```diff
@@ -1,37 +1,21 @@
-                    </SwiperSlide>
-                ))}
-            </Swiper>
+                <SliderGridColumn sliderId={sliderId} columnSize={[0, 4]}/>
+                <SliderGridColumn sliderId={sliderId} columnSize={[5, 9]}/>
+            </div>
```
This ranges use only here? If not you must declare it as a constants

ShadowZilot commented on src/components/common/SliderGridColumn/SliderGridColumn.tsx:
```diff
@@ -0,0 +1,47 @@
+                                    className="grid-slider__movie-release-date center-column-flex"
+                                >
+                                    <span
+                                        className="grid-slider__movie-release-date-number">{dayjs(data.releaseDate).format('DD')}</span>
+                                    <span
+                                        className="grid-slider__movie-release-date-month">{dayjs(data.releaseDate).format('MMM')}</span>
+                                </div>
+                                <button className="grid-slider__movie-watch-list-button"></button>
```
To avoid format date in place better transform data in map

ShadowZilot commented on src/components/layout/AppContent/AppContent.tsx:
```diff
@@ -84,7 +84,7 @@ export const AppContent = () => {
             >
                 <div
-                    className="content-container__movie-series-content"
+                    className={`content-container__movie-series-content ${index === 4 || index === 5 ? "start-row-center-flex" : ""}`}
```
Maybe we can use approach from this article https://habr.com/ru/articles/649381/

AlekseiKolesnikov committed:
1. Made NavBar stick to the top.
2. Fixed useSliderResponse.ts object, there are no repetitive objects anymore. Also, date reformating in the place where I receive it, not in tsx.
3. Fixed className conditions in the tsx by using clsx
4. Fixed right padding in each movie slider. There is no space between the edge of the screen and poster
5. Fixed the problem with touch of icons. They used to hoover on touch
